build/AzurePipelinesTemplates/WindowsAppSDK-BuildMRT-Steps.yml

```diff
@@ -21,6 +21,9 @@ steps:
     compilerPackageName: $(compilerOverridePackageName)
     compilerPackageVersion: $(compilerOverridePackageVersion)
     slnDirectory: $(Build.SourcesDirectory)\dev\MRTCore\mrt
+    includeUCRT: true
+    ucrtFeedPat: $(System.AccessToken)
+    platform: $(buildPlatform)
```

Comment:
this is the essential change - include the UCRT (STL/CRT libs/headers) along with the tools